climada/util/api_client.py

```diff
@@ -319,10 +329,10 @@ def get_dataset_by_uuid(self, uuid):
         NoResult
             if the uuid is not valid
         """
-        url = f'{self.host}/rest/dataset/{uuid}'
+        url = f'{self.url}/dataset/{uuid}'
         return DatasetInfo.from_json(Client._request_200(url))
 
-    def get_data_types(self, data_type_group=None):
+    def list_data_type_infos(self, data_type_group=None):
         """Returns all data types from the climada data API
```

Comment:
This returns more than the data types right? It returns the full info on all the data types no?

Reply:
right